lib/trader.js

```diff
@@ -407,6 +409,8 @@ Trader.prototype.pollHandler = function pollHandler (res) {
     this.triggers = createCompatTriggers(res)
   }
 
+  machineInfo.save(this.dataPath, res.machineInfo)
```

Comment:
Shouldn't we be using ``await`` here?

Reply:
Machine info isn't required for the machine to work, so we can move past this without waiting. Also this is inside the polling function, waiting for FS work should be prevented.